locales/en.json

```diff
@@ -13,6 +13,7 @@
     "app_already_up_to_date": "{app:s} is already up-to-date",
     "app_argument_choice_invalid": "Use one of these choices '{choices:s}' for the argument '{name:s}'",
     "app_argument_invalid": "Pick a valid value for the argument '{name:s}': {error:s}",
```

Comment:
Please, remove this keys:

- `app_argument_invalid`
- `domains_available`
- `pattern_password_app`
- `users_available`

Reply:
I'm going to use them actually to make better error messages, just haven't found the energy yet.

Reply:
Normally done